spec/models/audit_spec.rb

```diff
@@ -40,7 +40,7 @@
       expect(audit.save).to be_falsey
     end
 
-    it "can not have line items that has quantity as zero" do
+    it "can have line items that has quantity as zero" do
```

Comment:
Should there be an additional test to ensure that -1 is invalid?

Reply:
@dorner there is currently a test to test that negative integers are invalid. It is using -10, id be happy to update it to -1 or make a separate test if that is what you would like to see :)